python/tests/plugins/lsp_server/providers/test_goto_definition_provider.py

```diff
@@ -108,56 +110,64 @@ async def test_handles_goto_definition_request_when_definition_in_different_docu
         new_test_document_content = f"{TEST_DOCUMENT_CONTENT}{YAML_DOCUMENT_SEPARATOR}{TEST_SERVICE_THREE.to_yaml()}"
         await self.write_document(TEST_DOCUMENT_NAME, new_test_document_content)
 
-        text_range, *_ = self.provider.get_ranges_containing_name(new_test_document_content, TEST_SCHEMA_C.name)
-        line = text_range.start.line
-        character = text_range.start.character
-        res: GotoDefinitionResponse = await self.goto_definition(TEST_DOCUMENT_NAME, line=line, character=character)
+        target_location = self.get_definition_location_of_name(new_test_document_content, TEST_SCHEMA_C.name)
+        res: GotoDefinitionResponse = await self.goto_definition(
+            TEST_DOCUMENT_NAME, line=target_location.range.start.line, character=target_location.range.start.character
+        )
 
-        definition_range, *_ = self.provider.get_ranges_containing_name(added_content, TEST_SCHEMA_C.name)
+        definition_location = self.get_definition_location_of_name(added_content, TEST_SCHEMA_C.name)
 
         location = res.get_location()
         self.assertIsNotNone(location)
-        self.assertEqual(location.uri, self.to_uri(added_document.file_name))
-        self.assertEqual(location.range.start, Position(line=definition_range.start.line, character=definition_range.start.character))
-        self.assertEqual(location.range.end, Position(line=definition_range.end.line, character=definition_range.end.character))
+        self.assertEqual(path.basename(location.uri), path.basename(added_document.file_name))
+        self.assertEqual(
+            location.range.start,
+            Position(line=definition_location.range.start.line, character=definition_location.range.start.character),
+        )
+        self.assertEqual(
+            location.range.end,
+            Position(line=definition_location.range.end.line, character=definition_location.range.end.character),
+        )
 
     async def test_handles_goto_definition_for_root_keys(self):
         await self.create_document("spec.aac", get_aac_spec_as_yaml())
 
         res: GotoDefinitionResponse = await self.goto_definition(TEST_DOCUMENT_NAME, line=0, character=1)
-        schema_definition_location, *_ = self.get_definition_location_at_position(TEST_DOCUMENT_NAME, line=0, character=1)
+        schema_definition_location = self.get_definition_location_at_position(TEST_DOCUMENT_NAME, line=0, character=1)[0]
```

Comment:
Similar to previous comments, I think destructuring is a cleaner approach.

Reply:
I agree. I was under the impression that the destructing was throwing an error one a one-element array, complaining about being unable to unpack all of the elements. I think I was mistaken about what was being returned after I had made some alterations. I've reverted back as it's working now.